hydra-configs-torchvision/requirements/dev.txt

```diff
@@ -0,0 +1,2 @@
+torch==1.6
```

Comment:
Pytorch is already at 1.8, need to catch up eventually.

Reply:
once the approach is stabilized and configen is a little more mature, I think we will be able to track with more agility